.gitignore

```diff
@@ -157,4 +157,8 @@ Temporary Items
 
 # tmp script files
 *_DeploymentConfig.json
-*_BuildConfig.json
\ No newline at end of file
+*_BuildConfig.json
+
+# param files / local files
+*.param
```

Comment:
Are you sure you want to be ignoring `.param` files?

Reply:
changed to track param files